fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/BiomeUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biomes.v1;
+
+import net.fabricmc.fabric.impl.biomes.BiomeLists;
+import net.minecraft.world.biome.Biome;
+
+public final class BiomeUtils
```

Comment:
`SpawnBiomes`

Reply:
This is not just for spawn biomes. This is for any code which is "common" among biome sources. Currently it is just spawn biomes though.

Reply:
`MoreBiomes` then, following Guava naming conventions for the scenario wherein a plural named class for the overall type already exists in the environment.

Reply:
yeet